tests/tests-skeletons/check-GeneralizedTime.c

```diff
@@ -190,8 +190,8 @@ check_fractions() {
 
 static void
 compare(int lineno, int cmp_control, const char *astr, const char *bstr) {
-    GeneralizedTime_t a = {(uint8_t *)strdup(astr), strlen(astr)};
-    GeneralizedTime_t b = {(uint8_t *)strdup(bstr), strlen(bstr)};
+    GeneralizedTime_t a = {(uint8_t *)strdup(astr), strlen(astr), {0}};
+    GeneralizedTime_t b = {(uint8_t *)strdup(bstr), strlen(bstr), {0}};
```

Comment:
Why is this? Some other warning? They're supposed to be 0-initialized on their own.

Reply:
Because of the clang https://travis-ci.org/vlm/asn1c/jobs/289782301#L1179-L1186
I'm done for today, I will continue tomorrow